frontend/package.json

```diff
@@ -100,6 +100,7 @@
     "eslint": "^7.29.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.1.4",
-    "prettier": "^2.2.1"
+    "prettier": "^2.2.1",
```

Comment:
I feel a better implementation would be to add in a .prettierrc.json file and asking everyone to install the extension on VSC. This is good for everyone in the long run when they go intern, as well as enforcing a standardised saving format

Reply:
agreed, need some man-effort in defining versions, rules etc. for ur consideration. rmb to reference correct path in package.json